Echo.json5

```diff
@@ -65,6 +65,7 @@
     { name: 'AREnableMyCollectionHFOnboarding', value: false },
     { name: 'ARReorderSWAArtworkSubmissionFlow', value: false },
     { name: 'AREnableNewOpaqueImageComponent', value: false },
+    { name: 'ARIosSearchTabCuratedCollections', value: true },
```

Comment:
**question**: do we need the `ios` prefix here? as far as I understand, this functionality will be available on android?

Reply:
I think it's better to make `false` by default